package.json

```diff
@@ -24,7 +24,8 @@
   "types": "./types/index.d.ts",
   "scripts": {
     "mocha": "mocha",
-    "lint": "eslint -c .eslintrc.js src/*.js src/lib/*.js",
+    "lint": "eslint src/*.js src/lib/*.js src/plugins/*.js",
```

Comment:
Does this also need to revert since we switched back to JS or was the `-c` implied all along and I'm just being too explicit?

Reply:
Yes, by default ESLint searches for a config file named `.eslintrc` (with various endings).

See https://eslint.org/docs/user-guide/configuring#using-configuration-files-1